packages/babel-preset-env/src/targets-parser.js

```diff
@@ -194,35 +177,36 @@ const getTargets = (targets: Object = {}, options: Object = {}): Targets => {
   }
 
   // Parse browsers target via browserslist
-  const browsersquery = validateBrowsers(targets.browsers);
+  let browsersquery = validateBrowsers(targets.browsers);
 
   const hasTargets = Object.keys(targets).length > 0;
   const shouldParseBrowsers = !!targets.browsers;
   const shouldSearchForConfig =
     !options.ignoreBrowserslistConfig && !hasTargets;
 
-  if (shouldParseBrowsers || shouldSearchForConfig) {
-    // If no targets are passed, we need to overwrite browserslist's defaults
-    // so that we enable all transforms (acting like the now deprecated
-    // preset-latest).
-    //
-    // Note, if browserslist resolves the config (ex. package.json), then usage
-    // of `defaults` in queries will be different since we don't want to break
-    // the behavior of "no targets is the same as preset-latest".
-    if (!hasTargets) {
-      browserslist.defaults = objectToBrowserslist(targets);
+  if (!hasTargets) {
+    // If no targets are passed, we call loadConfig to resolve any external browserslist
+    // config.
+    if (shouldSearchForConfig) {
+      browsersquery = browserslist.loadConfig({
+        path: options.configPath,
+      });
     }
+    // If there is no external browserslist config and no targets are specified,
+    // we return an empty array so that "no targets and no browserslist is the same as
+    // preset-latest"
+    if (!browsersquery) {
+      browsersquery = [];
+    }
+  }
 
+  if (shouldParseBrowsers || shouldSearchForConfig) {
     const browsers = browserslist(browsersquery, {
-      path: options.configPath,
```

Comment:
Because `browsersquery` is never `undefined` here, `browserslists` will never `loadConfig` and therefore we don't need to pass `path` option here.